shodan/stream.py

```diff
@@ -27,9 +27,12 @@ def _create_stream(self, name, timeout=None):
         # If the user requested a timeout then we need to disable heartbeat messages
         # which are intended to keep stream connections alive even if there isn't any data
         # flowing through.
-        if timeout:
+        if timeout is not None:
```

Comment:
That changes the behaviour if `timeout = False` in an unintuitive way.

Reply:
Compare to None should be `is` or `is not` according to PEP 8 best practice, this value should be either `None` or a number, it shouldn't be a boolean value

Reply:
This is a potentially breaking change. I agree that it would be better if we made it clear that it should be ``None`` or a number but that's not how it works at the moment. I think there's a conversation to be had about creating a Python3 version that uses type hints and properly conforms to PEP8 though that's outside the scope of what this MR is about.